pkg/volume/iscsi/iscsi.go

```diff
@@ -198,6 +199,7 @@ type iscsiDisk struct {
 	chap_discovery bool
 	chap_session   bool
 	secret         map[string]string
+	InitiatorName  string
```

Comment:
Comment from @rootfs, I've added initiatorname check before deleting iface at in DetachDisk().
Therefore, this should start with uppercase("InitiatorName") to store the value into jsonfile.